packages/back-end/src/api/openapi/openapi.yaml

```diff
@@ -190,6 +193,16 @@ paths:
   /organizations/{id}:
     put:
       $ref: "./paths/putOrganization.yaml"
+  /organizations/{id}/environments:
+    get:
+      $ref: "./paths/listEnvironments.yaml"
+    post:
+      $ref: "./paths/postEnvironment.yaml"
+  /organizations/{id}/environments/${environmentId}:
+    put:
+      $ref: "./paths/putEnvironment.yaml"
+    delete:
+      $ref: "./paths/deleteEnvironment.yaml"
```

Comment:
Don't put these under `/organizations/{id}`.  The routes should be `/environments/` and `/environments/${id}`